docs/comands-usage.md

```diff
@@ -0,0 +1,112 @@
+# Azure CAPI CLI Usage
+
+See [Kubernetes Cluster API Provider Azure](https://capz.sigs.k8s.io/) for more information.
```

Comment:
I think this sentence is confusing here. The document is about "az capi" usage, but we start out by pointing to a source of more general information about CAPZ which won't say anything about "az capi."

Maybe this should go near the end of the document under a section called something like "Learn more about the Cluster API Provider for Azure."

An introductory sentence should probably summarize what's to come in the rest of the document. Maybe:

"This document summarizes `az capi` commands and their arguments."